components/navigable-container/index.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * External Dependencies
+ */
+import { omit, noop } from 'lodash';
+
+/**
+ * WordPress Dependencies
+ */
+import { Component } from '@wordpress/element';
+import { focus, keycodes } from '@wordpress/utils';
+
+/**
+ * Module Constants
+ */
+const { UP, DOWN, LEFT, RIGHT, TAB } = keycodes;
+
+function cycleValue( value, total, offset ) {
+	const nextValue = value + offset;
+	if ( nextValue < 0 ) {
+		return total + nextValue;
+	} else if ( nextValue >= total ) {
+		return nextValue - total;
+	}
+
+	return nextValue;
+}
+
+class NavigableContainer extends Component {
+	constructor() {
+		super( ...arguments );
+		this.bindContainer = this.bindContainer.bind( this );
+		this.onKeyDown = this.onKeyDown.bind( this );
+
+		this.getFocusableContext = this.getFocusableContext.bind( this );
+		this.getFocusableIndex = this.getFocusableIndex.bind( this );
+	}
+
+	bindContainer( ref ) {
+		this.container = ref;
+	}
+
+	getFocusableContext( target ) {
+		const { deep = false, onlyBrowserTabstops } = this.props;
+		const finder = onlyBrowserTabstops ? focus.tabbable : focus.focusable;
+		const focusables = finder
+			.find( this.container )
```

Comment:
`Array#find` is not polyfilled and is not supported by IE11. This results in crashes in IE11 (e.g. opening inserter).

Reply:
Nevermind, this is the focus utility `find` method 🤦‍♂️ There's still an error, but I think it's caused by https://github.com/WordPress/gutenberg/pull/3281/files#r151239536.